app-misc/mosquitto/mosquitto-1.4.15.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+inherit systemd user toolchain-funcs python-any-r1
+
+DESCRIPTION="An Open Source MQTT v3 Broker"
+HOMEPAGE="http://mosquitto.org/"
+SRC_URI="http://mosquitto.org/files/source/${P}.tar.gz"
+LICENSE="EPL-1.0"
```

Comment:
Empty line above it. What is the source of this bad template a lot of people seem to be using?

Reply:
I just copied the old ebuild and renamed it via `mv`.

Reply:
Between `SRC_URI` and `LICENSE`.